framework/src/bundle/BundleContext.cpp

```diff
@@ -203,7 +203,7 @@ namespace cppmicroservices
         return bus;
     }
 
-    ServiceRegistrationU
+    [[nodiscard]] ServiceRegistrationU
```

Comment:
I think `[[nodiscard]]` can be removed here, it only needs to be in the function declaration.